### PR TITLE
[Mobile app] Updating dependency packages

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)

- Updates dependency packages

#### Status of maturity (all need to be checked before merging):

- [ ] I've tested this locally
- [ ] I consider this code done
- [ ] This change ready to hit production in its current state
- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- The app should run as expected without any errors.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

### CODE REVIEW

File: mobile/pubspec.yaml [hunk 1]
```diff
@@ -27,70 +27,67 @@ dependencies:
...
   geolocator: ^9.0.2
-  flutter_local_notifications: ^12.0.3
+  flutter_local_notifications: ^13.0.0
   url_launcher: ^6.1.6
   path_provider: ^2.0.11
   animations: ^2.0.7
-  provider: ^6.0.4
-  charts_flutter: ^0.12.0
+  provider: ^6.0.5
+  charts_flutter: ^0.12.0 # TODO: Use another package. charts_flutter is discontinued. https://pub.dev/packages/charts_flutter
```
Comment (NoahNsimbe) [inline mobile/pubspec.yaml:40]: @BenjaminSsempala @Mnoble-19 take note of this.

File: mobile/lib/screens/web_view_page.dart [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
-import 'dart:io';
-
 import 'package:app/themes/theme.dart';
 import 'package:app/utils/utils.dart';
 import 'package:app/widgets/widgets.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:webview_flutter_android/webview_flutter_android.dart';
+import 'package:webview_flutter_wkwebview/webview_flutter_wkwebview.dart';
 import 'package:webview_flutter/webview_flutter.dart';
```
Comment (NoahNsimbe) [inline mobile/lib/screens/web_view_page.dart:8]: incorporating webview migration changes

File: mobile/pubspec.yaml (hunk 1 above)
Comment (BenjaminSsempala) [reply mobile/pubspec.yaml:40]: @NoahNsimbe do you already have a replacement in mind?

**Review by Baalmart — APPROVED** [review]
Thanks @NoahNsimbe , this looks good to me.
I have just left one review comment for your consideration.

I will wait for @Mnoble-19 and @BenjaminSsempala to run the PRs and review before I undertake the merge duty...

File: mobile/ios/Runner.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -10,6 +10,7 @@
 		1498D2341E8E89220040F4C2 /* GeneratedPluginRegistrant.m in Sources */ = {isa = PBXBuildFile; fileRef = 1498D2331E8E89220040F4C2 /* GeneratedPluginRegistrant.m */; };
 		3B3967161E833CAA004F5970 /* AppFrameworkInfo.plist in Resources */ = {isa = PBXBuildFile; fileRef = 3B3967151E833CAA004F5970 /* AppFrameworkInfo.plist */; };
 		74858FAF1ED2DC5600515810 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 74858FAE1ED2DC5600515810 /* AppDelegate.swift */; };
+		80A0C01042445476F75F2EA8 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 1506EDD7307202CBEC6D040C /* GoogleService-Info.plist */; };
```
[the remark this answers is not in the document]
Comment (NoahNsimbe) [reply mobile/ios/Runner.xcodeproj/project.pbxproj:13]: Am not sure @Baalmart but think they should be in version control. If they were not supposed to be in version control, I believe the flutter cli command for creating projects `flutter create airqo` would have added them to the `.gitignore` file.

**Review by BenjaminSsempala — APPROVED** [review]
Tested out and Works well for me

File: mobile/ios/Runner.xcodeproj/project.pbxproj (hunk 3 above)
[the remark this answers is not in the document]
Comment (Baalmart) [reply mobile/ios/Runner.xcodeproj/project.pbxproj:13]: Alright @NoahNsimbe , thanks for the clarification

**Pull request merged**